### DISCUSSION TITLE
No Pub Key for Repo

### DISCUSSION THREAD

**Discussion by Nippler1** [Security-Onion-Solutions/securityonion#11537] [Unsupported Versions]
I'm trying to install SO Version 2.3.270 on ubuntu 2004. Unfortunately there is no pub key for the following repository:
https://repo.securityonion.net/file/securityonion-repo/ubuntu/20.04/amd64/salt3004.2
How can I download the key manually - or is there really none?

Thank you very much in advance.

**Comment by dougburks** [comment]
Security Onion 2.3 will reach End Of Life in less than 6 months:
https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html

So if you're performing a new installation, we'd recommend going with 2.4 instead:
https://blog.securityonion.net/2023/10/security-onion-2420-now-available.html
